tests/test_clip_inference/test_mapper.py

```diff
@@ -1,10 +1,12 @@
+import pytest
 import pickle
 import os
+
 from clip_retrieval.clip_inference.mapper import ClipMapper
-import os
 
 
-def test_mapper():
+@pytest.mark.parametrize("model", ["ViT-B/32", "open_clip:ViT-B-32-quickgelu"])
+def test_mapper(model):
```

Comment:
You need to pass that variable down there

Reply:
oops